contracts/modules/Checkpoint/ERC20/ERC20DividendCheckpoint.sol

```diff
@@ -257,7 +257,7 @@ contract ERC20DividendCheckpoint is ERC20DividendCheckpointStorage, DividendChec
         dividends[_dividendIndex].reclaimed = true;
         Dividend storage dividend = dividends[_dividendIndex];
         uint256 remainingAmount = dividend.amount.sub(dividend.claimedAmount);
-        require(IERC20(dividendTokens[_dividendIndex]).transfer(wallet, remainingAmount), "transfer failed");
+        require(IERC20(dividendTokens[_dividendIndex]).transfer(getTreasuryWallet(), remainingAmount), "transfer failed");
```

Comment:
For good measure do you think we should be checking that `getTreasuryWallet()` doesn't return address(0)? I agree that with the current code it can't, but in some future model if there was a bug, perhaps the datastore treasury wallet could be over-written with 0, and we wouldn't want to send funds there under any circumstances.

Reply:
on this measure, global treasury wallet is only be changed using the `changeTreasuryWallet()` function that is present in the SecurityToken and we have the 0 address check init.  So it doesn't require any further check.  And yes there is no harm to add one more check.